shared/build.gradle.kts

```diff
@@ -35,21 +35,25 @@ kotlin {
                 implementation(compose.runtime)
                 implementation(compose.foundation)
                 implementation(compose.material)
-                @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
-                implementation(compose.components.resources)
+                api("dev.icerock.moko:resources:0.20.1")
```

Comment:
version should be from gradle.properties as we use same way that in original template

Reply:
agree, changed